.github/workflows/ci-publish.yaml

```diff
@@ -0,0 +1,47 @@
+name: "Publish Docs"
+
+on:
+  workflow_run:
+    workflows: ["CI"]
+    branches: [master]
+    types: 
+      - completed
+
+jobs:
+  publishing:
+    name: Publish Documentation
+    runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    steps:
+    - name: 📥 Download stable docs with released
+      uses: dawidd6/action-download-artifact@v2
+      with:
+        workflow: release.yaml
+        workflow_conclusion: success
+        name: stable
+        path: docs/public/
+
+    - name: 📥 Download & merge unstable docs with released
+      uses: dawidd6/action-download-artifact@v2
+      with:
+        workflow: ci.yaml
+        workflow_conclusion: success
+        name: unstable
+        path: docs/public/unstable
+
+    - name: 💾 Upload artifact
+      uses: actions/upload-artifact@v3
+      with:
+        name: head-protocol
+        path: docs/public
```

Comment:
I would remove this as it is my understanding that it only re-upload some artifact it downloaded before. If it is needed, I would add an explanation of why we need to do that.

Reply:
this artifact is the result of merging the latest release with master.

we agreed during tactical these artifacts were valuable for debugging purposes.

do you think they are not needed anymore?

Reply:
I don't think they're valuable anymore but that's your call 👍